docs/release.md

```diff
@@ -0,0 +1,105 @@
+## Release process
+
+To release a new version, you need to be a member of `gcp-variant-transforms`,
+and `gcp-variant-transforms-test` projects and have write access to the GitHub
+repo.
+
+The process is as follows:
+
+1. Navigate to
+   [Container Builder build history](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   and verifly that the latest integraion tests have passed.
+1. Choose the appropriate version number by following the semantics in
+   https://semver.org/. See the **Summary** section for a quick overview.
+1. Update the version number in
+   [setup.py](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/setup.py)
+   by submitting a change. Please note the commit SHA generated by GitHub (it's
+   a long string like `4ff1fdfb3957a115a80fe9343306952cd98b6e5c`). You may run
+   the following command to make the next steps easier:
+   `export COMMIT_SHA=<commit sha>`.
+1. Wait until the
+   [Cloud Build](https://console.cloud.google.com/gcr/builds?project=gcp-variant-transforms-test)
+   process builds a new image that contains the change submitted in the previous
+   step. The image has the form
+   `gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:${COMMIT_SHA}`.
+1. The tests in the previous step only cover presubmit tests. We require the
+   release image to pass all tests including `huge_tests`. Run the following on
+   your workstation in a **clean** branch synced to the version change commit
+   and replace `${COMMIT_SHA}` with the appropriate value from the previous
+   step.
+
+   ```bash
+   # To create a clean branch. Skip if you already have one.
+   temp_dir="$(mktemp -d)"
+   cd "${temp_dir}"
+   git clone https://github.com/googlegenomics/gcp-variant-transforms.git
+   cd gcp-variant-transforms
+   git checkout "${COMMIT_SHA}"
+   # Run tests.
+   ./deploy_and_run_tests.sh --image_tag ${COMMIT_SHA} \
+       --keep_image --skip_build --run_all_tests \
+       --project gcp-variant-transforms-test
+   ```
+1. Wait until the test is done. It will take about 2 hours. Ensure that there
+   are no failures. In case of failures, rollback the version change in
+   `setup.py` and continue from Step #1 once the errors are fixed.
+1. To update the docker image, you can either:
+   * (**Preferred**) Copy the verified image to the main project by running the
+     following on your workstation. Note that you require
+     [docker](https://www.docker.com/) to be installed.
+
+     ```bash
+     gcloud auth configure-docker
+     SOURCE_IMAGE=gcr.io/gcp-variant-transforms-test/gcp-variant-transforms:"${COMMIT_SHA}"
+     TARGET_IMAGE=gcr.io/gcp-variant-transforms/gcp-variant-transforms:"${COMMIT_SHA}"
+     docker pull "${SOURCE_IMAGE}"
+     docker tag "${SOURCE_IMAGE}" "${TARGET_IMAGE}"
+     docker push "${TARGET_IMAGE}"
+     ```
+
+   * Push a new image from source by running:
```

Comment:
Why do you even document this option? Is it because the previous step relies on `docker` being installed? IMO only an image that has been fully tested should be released (I understand the next step reproduces the same image but it is technically a new image).

Reply:
Good point. Removed :) And yes, I wanted to provide an option for a user who does't have docker installed, but I think it's ok to require it.